Resources/doc/deployment/deployment.md

```diff
@@ -1,133 +1,96 @@
 # Using deployment
 
-We use Capifony as a base, but we extended it with our own specific
-configuration for usage with the framework.
+## Install deployer
 
-All configuration for the specific stage (staging, production) is done in the
-`deployments/stages/`-folder, wherein each stage has its own file.
+To use the deployer we need to have a local installation of the deployer.
+Run the following commands in the terminal:
 
-General configuration is done in the Capfile.
+```
+curl -LO https://deployer.org/deployer.phar
+mv deployer.phar /usr/local/bin/dep
+chmod +x /usr/local/bin/dep
+```
 
-## Configuration
+The deploy settings can be found in the deploy.php file.
 
-### Capfile
+## Configuration
 
-When you create a project you will need to fill in 3 variables:
+You will need to fill in 4 variables:
 
 * `:client`, the `xxx` should be replaced with the name of the client.
 * `:project`, the `xxx` should be replaced with the name of the project.
 * `:repository`, the `xxx` should be replaced with the url of the git-repo.
+* `:production_url`, the `xxx` should be replaced with the production url.
 
 Remark: when choosing a name for the project, please don't use generic names
 as: site, app, ... as it makes no sense when there are multiple projects for
 that client.
 
-### deployments/stages/staging.rb
-
-In the is file you can configure which branch should be deployed on the
-staging-server. By default this is the staging-branch
-
-* `:branch`, if needed you can replace `staging` with your branch.
-
-Remark: when you change this, make sure you notify other people, as when they
-deploy and are not working on your branch your changes will be lost. Also make
-sure that you change it back before creating a Pull Request.
-
-### deployments/stages/production.rb
-
-In the is file you can configure production specific items. In most cases the
-server-layout will be different from the staging-server, so there is somewhat
-more to configure.
-
-The default production.rb is inspired on the Hostbots-server-layout.
+### Define staging server
```

Comment:
"a staging server" or "the staging server"

Reply:
771ee017859d1412017ef62ece84d5c6a39d6be6